frontend/src/Preferences.vue

```diff
@@ -101,7 +101,7 @@ export default {
       videocodec: '',
       agc: false,
       defaultHdVideo: false,
-      conferenceRoomType: 'MESH',
+      conferenceRoomType: null,
```

Comment:
That's fine to initialize it to null here, but you also need to set it in the 'created' lifecycle hook based on the store, otherwise the dropdown will be blank even though the store has Mesh.

Setting it in the created hook is really all that was needed for this story as it turns out.